src/modules/askForNumber.js

```diff
@@ -0,0 +1,38 @@
+'use strict';
+
+const isNumberValid = require('./isNumberValid');
+const readline = require('readline');
+const getBullsCows = require('./getBullsCows');
+
+const terminal = readline.createInterface({
+  input: process.stdin,
+  output: process.stdout,
+});
```

Comment:
<img width="811" alt="image" src="https://github.com/mate-academy/node_bulls-and-cows/assets/86789161/c22ebd80-e030-4372-9a96-e59606c5f340">